### ISSUE TITLE
`winutil` desktop tweaks results in People icon and Search box on taskbar.

### ISSUE THREAD

**Issue by Obegg** [ChrisTitusTech/winutil#1867]
**Describe the bug**
`winutil` desktop tweaks results in People icon and Search box on taskbar.

**To Reproduce**
```
(irm https://raw.githubusercontent.com/ChrisTitusTech/winutil/main/winutil.ps1) -replace 'function Install-WinUtilChoco {','function Install-WinUtilChoco {return' | iex
```
1. Go to 'Tweaks'
3. Click on 'Desktop'
4. Click on 'Run Tweaks'
5. Restart PC

**Expected behavior**
No errors.
People icon on taskbar won't show.
Search box on taskbar won't show.

**Actual behavior**
Errors.
People icon on taskbar show.
Search box on taskbar show.

**Screenshots**
![image](https://github.com/ChrisTitusTech/winutil/assets/13978119/63b7eb6f-e7bd-4f6f-a013-98f8cc4c966b)
![image](https://github.com/ChrisTitusTech/winutil/assets/13978119/f097572f-e626-4963-be2c-212fd64a226e)
![image](https://github.com/ChrisTitusTech/winutil/assets/13978119/14d2a024-d8ca-4121-beec-f939e92c891f)

**Comment by fr0stedfl4ke** [comment]
I had the same issue

**Comment by og-mrk** [comment]
Hello @Obegg
I can confirm that these concerns of yours are noted once `OO Shutup` got fixed in the latest update of WinUtil, and I do agree on how annoying it can be, but when running tweaks (specifically the `OO Shutup Tweak`), it'll basically remove Microsoft's Spyware by changing a bunch of system polices, and so it'll have some System/User Preferences get changed to Windows Default Values, namely the Windows Search on the Taskbar being shown, and the Widgets, among others as well

besides the Visual Changes, Some Users that run WinUtil, which're used to Specific Features that come with Windows 10/11, _namely Windows Clipboard History Feature_, are presented by the fact `OO Shutup Tweak` disables Clipboard By default, see [this comment](https://github.com/ChrisTitusTech/winutil/issues/1871#issuecomment-2074847855) to know how to re-enable it.

Future patches/changes will fix this issue, lets hope we get it soon 🤞
for the time being, you'll have to re-do some Customization manually, which's somewhat of a tedious process if you're do a lot of Windows Installs, or (re)installing windows on a regular basis.

**Comment by og-mrk** [comment]
This was fixed by removing the OO Shutup Tweak (in PR #2099), and only leaving `Run OO Shutup 10` Button so you can select/apply anything that O&O Shutup has to offer.
